apps/android/app/src/main/res/values-de/strings.xml

```diff
@@ -358,15 +358,15 @@
   <string name="how_to_use_simplex_chat">Wie man SimpleX nutzt</string>
   <string name="markdown_help">Markdown Hilfe</string>
   <string name="markdown_in_messages">Markdown in Nachrichten</string>
-  <string name="chat_with_the_founder">Senden Sie Fragen und Ideen</string>
+  <string name="chat_with_the_founder">Senden Sie Ihre Fragen und Ideen</string>
```

Comment:
This is a list item inside the settings menu, can it be shortened? It may not fit into one line there, might be better to sacrifice some grammar than to have it go two lines.